### DISCUSSION TITLE
Can't open start menu

### DISCUSSION THREAD

**Discussion by cinematic6436** [valinet/ExplorerPatcher#704] [Q&A] [answered]
I just upgraded from Windows 10 to Windows 11 today. I downloaded & ran ep_setup.exe as administrator. No prompt for elevation or any indication it was installing -- it just dumped me to the desktop and reloaded what programs I had open (I believe Firefox, possibly Notepad and Explorer). 

I then right clicked on the taskbar & selected "properties." For format, I selected Windows 10. Now I have a Windows 10 start icon but can't open the start menu. I click on it, and the highlighted button just gets a few shades darker. Nothing opens. "Properties" is no longer an option when I right click the task bar. I tried uninstalling & reinstalling Explorer Patcher, but nothing changes.

I had 7+ Taskbar Tweaker installed with no problem in Windows 10, but after the upgrade to 11, I got an error message saying it couldn't start, possibly because it "wasn't compatible with this version of Windows." I uninstalled it and installed the latest version (5.12.3) but the problem remains. Also, the clock on the far right is missing. There doesn't appear to be an option to enable/disable it in the settings.

Edition	Windows 11 Pro
Version	21H2
Installed on	‎1/‎20/‎2022
OS build	22000.438

Booted into safe mode, and results are the same. I'm using he latest version of the setup program as of 01/20/2022. Any help would be greatly appreciated.

**Comment by Hr46ph** [comment]
I am experiencing the same issue on Windows 11 pro insider preview build 22538, 220114-1500.

Win-key no longer opens the start menu, clicking it doesn't do anything. Choosing Windows 10 or Windows 11 style doesn't change. Only uninstalling and reverting back to Windows 11 seems to resolve the issue.

**Reply by valinet** [reply]
Windows Insider builds are not supported. No one tests and *develops* against them, Microsoft does all kinds of funky changes, this is kind of expected. It’s also mentioned in the wiki.

**Comment by cinematic6436** [comment, marked as answer]
I gave up & rolled back to Windows 10, then re-upgraded to Windows 11. Installed ep_setup.exe again, and now everything's working as it should. Zero idea what went wrong the first time.

**Reply by amel-fit** [reply]
I found a solution. It is probably due to Winaero Tweaker (at least it was in my case)
There was an option called "Classic Taskbar" which you could enable and additionally install Explorer Patcher.
That setting is no more, but for the people that enabled it in old builds:
It created a registry value that you have to delete (always back up everything just in case). To quote their website:
Navigate to the following key: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell\Update\Packages. You can copy this path and paste it into the address bar in the Registry Editor.
There should be an entry there called "UndockingDisabled"
Delete that entry. That's that
In case it isn't there, you're SOL as far as I know :/

**Comment by vp-bct** [comment]
I'm facing the same issue here with Windows 11 Pro 21H2 build 22000.469. Start button does not respond. Neither does the Win key. Uninstalling ExplorerPatcher reverts the normal Windows 11 Start button behavior. Unfortunately I'm unable to repeat the upgrade. Any possible workaround instead of repeating upgrade?

**Reply by Khdoop** [reply]
+1 same issue here
cant open start menu (both windows key or the icon)
cant open search
cant open cortana
tried almost all settings, nothing helped
tried with 2 versions of the patcher 22000.469.41.16 and 22000.493.42.11, both NOT working
OS build 22000.493


edit:
changing taskbar mode to windows 11 is NOT working and doesnt do anything
uninstalling the patcher is NOT reverting everything that it has changed, so im stuck to not working start menu
i feel like my only option is to downgrade to win10 at this point, not sure what else to try
@valinet

**Reply by GSquared** [reply]
@Khdoop Same issue here, installed it (since it had zero confirmation to install), promptly uninstalled it and now on reboot nothing functions anymore just as you stated.

Would highly advise anyone browsing this against installing this in its current state.

If you found a fix for this, please let me know as it essentially busted the entire W11 start menu/task bar.

**Reply by Khdoop** [reply]
i downgraded back to win10 as this was my only option to have a working taskbar...

**Reply by valinet** [reply]
> Same issue here, installed it (since it had zero confirmation to install)

How would a prompt to confirm the installation have helped in your case? Why does a prompt to confirm the installation make any sense considering that you deliberately double clicked the *installer* and chose to run it. 99.9999% you still would have clicked “Yes” without setting a restore point or making a backup first, despite being recommended to do so on every software install actually. That was also a big factor in why you had to do a reinstall or whatever when things went wrong.

As for the rest of the attitude in your post, can hardly make any comment. A lot of people have this working just fine. You *may* have encountered some problem related to this, you came to the portal where development of this app takes place, and instead of helping out identify and fix this supposed bug, your recommendation is to have no one use this anymore. Then how should it be fixed? You expect software to fix itself, grow a consciousness and repair itself when it sees no one uses it anymore or what…?

As I said, I am starting to question more and more the usefulness and idea behind this forums day after day. Please consider the difference between the purpose of a community around a project on GitHub and a review board on Google Maps… if not obvious already, apparently.

As for the supposed “bug” itself, yeah, no insight on it from you, I can’t magically fix it, I am not a wizard, I say this *again*.

**Comment by TimQSO** [comment]
I'm having the same problem. Installed the patch. My open windows were ungrouped, but the Start button didn't do anything. Uninstalled the patch. The open windows were grouped again, but still no working start menu. Reinstalled again, no luck. Tried some settings, no luck.

I've installed Start11 from Stardock in the mean time to fix the issue.

**Reply by valinet** [reply]
Check to see if there is a dxgi.dll file in C:\Windows \SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy and delete it if it’s there. If not, it’s not an EP problem.

**Reply by TimQSO** [reply]
There is only a dxgi.prev file there no dll is present.

It is not really fair to say that it isn't an EP problem. This is a fairly new and clean W11 installation.

While Start11 helps me to continue to work on my PC, it breaks again when I remove Start11.

**Reply by valinet** [reply]
Okay, so EP was successfully uninstalled. If EP is not there, how is it an EP problem…?

**Reply by TimQSO** [reply]
Maybe because before the EP installation my Start menu was working? And after EP installation it stopped working and it keeps not working after removing EP.

Something has changed because of EP and the change breaks it even after removal of EP.

Maybe some registry settings?

Furtermore, you see that other users have the same issue and the issue persists after removing EP.

**Reply by valinet** [reply]
If you want to learn how the program works, then I told you, the installer places a file in that folder so EP injects the Start menu executable. It does a similar thing in C:\Windows \SystemApps\ShellExperienceHost_cw5n1h2txyewy and C:\Windows. If no dxgi.dll files are there, then the program is uninstalled and it cannot actively affect the other things on the system.

Regarding registry entries, idk, unless you changed something, it generally doesn’t set random things up. But again, idk.

If there is a problem, best is to try to diagnose and fix it, I don’t know each and every thing in the world, it’s irrelevant if others experience something that looks similar or not, what’s relevant is to actually fix it.

Also, maybe delete the HKEY_CURRENT_USER\Control Panel\Desktop\WindowMetrics\MinWidth entry, if there.

**Reply by TimQSO** [reply]
I've tried several things without luck. But I was lucky, my son installed Team Speak yesterday and during the installation of Team Speak a Recovery Point was made. 

I used a restore to this Recovery Point and all is working like a charm again.

I think the issue is somewhere in the Search / Search index or something. It causes to block the Start Menu and all search functions. The search bar in Windows Explorer was also not working anymore.

However resetting Search, Rebuilding and so on was not working. Also reinstalling core functions via PowerShell was not working either. 

So I don't know what caused the issue, but I'm happy that it was solved by using a recovery point.

**Reply by valinet** [reply]
I am glad that you got it solved. And I tell you, I have no intention with EP of breaking stuff, everything bad that happens, if it really happens because of EP, is an accident (i.e. not intentional, but coding mistakes, oversights etc on my part) and I strive to correct those as soon as possible.

**Comment by TimQSO** [comment]
I dug into this and it is not an issue with EP. I've done it now with only registry settings and the same issue occurs.

The problem is in the setting:

HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell\Update\Packages
With DWORD""UndockingDisabled"

Set this to 0 and your menu will work again. However in the task bar grouping is then still active.

**Reply by valinet** [reply]
Yeah, and advice for future users thinking whether to do this: STOP using that setting!!! The whole point of EP and why it was made initially was so that users don't have to enable `UndockingDisabled`. That breaks a lot of things and doesn't really work well at all. To be honest, I don't even understand why tools like Winaero Tweaker suggest that to users, it can break more things than actually fix.

If you want the Windows 10 taskbar in 11, ExplorerPatcher is an alternative. There are others, like StartAllback, for example.

Thanks
